Directory.Build.props

```diff
@@ -1,4 +1,10 @@
 ﻿<Project>
+  <PropertyGroup>
+    <EnableNETAnalyzers>true</EnableNETAnalyzers>
+    <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
+    <AnalysisLevel>latest-Recommended</AnalysisLevel>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
I like dotnet/msbuild style of it: do not apply it in `props` but add in cmd build script. However, since we don't have any scripts atm this seems to be the only possible gatekeeper.